src/query/expression/src/schema.rs

```diff
@@ -557,6 +557,7 @@ impl TableSchema {
                         )
                     });
                 }
+                Scalar::Map(_) | Scalar::Array(_) => {}
                 _ => {
```

Comment:
Prefer to remove '_,' give all the case, we will never missed anything.

Reply:
Scalar is a long enum, added `debug_assert!(!default_value.is_nested_scalar());`